.changeset/yellow-eels-stare.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
The contents are missing here

Reply:
Yep. See note in PR description.

Reply:
I wonder if Changesets can follow Conventional Commits which has the convention of using `chore:` or `refactor:` prefix for this case.

https://www.conventionalcommits.org/en/v1.0.0/#:~:text=types%20other%20than%20fix%3A%20and%20feat%3A%20are%20allowed